ecosystem/sep-0010.md

```diff
@@ -112,19 +112,19 @@ GET https://auth.example.com/?account=GCIBUCGPOHWMMMFPFTDWBSVHQRT4DIBJ7AD6BZJYDI
 On success the endpoint must return `200 OK` HTTP status code and a JSON object with these fields:
 
 * `transaction`: an XDR-encoded Stellar transaction with the following:
-  * source account set to server's signing account
+  * source account set to the **Home Domain**'s ``SIGNING_KEY``
   * invalid sequence number (set to 0) so the transaction cannot be run on the Stellar network
-  * time bounds: `{min: now(), max: now() + 900 }` (we recommend expiration of 15 minutes to give user time to sign transaction)
+  * time bounds: `{min: now(), max: now() + 900 }` (we recommend expiration of 15 minutes to give the **Client Application** time to sign transaction)
   * operations:
-    * `manage_data(source: client_account, key: '<home domain> auth', value: random_nonce())`
-      * The value of key is the home domain the client is authenticating with, followed by `auth`. It can be at most 64 characters.
+    * `manage_data(source: authenticating_account, key: '<home domain> auth', value: random_nonce())`
+      * The value of key is the **Home Domain**, followed by `auth`. It can be at most 64 characters.
       * The value must be 64 bytes long. It contains a 48 byte cryptographic-quality random string encoded using base64 (for a total of 64 bytes after encoding).
     * subsequent operations, order unimportant:
       * `manage_data(source: server_account, key: 'web_auth_domain', value: web_auth_domain)`
-        * The value is the web auth domain, the domain hosting the web auth endpoint and that the client connected to when requesting the challenge transaction. It can be at most 64 characters.
+        * The source account is the **Home Domain**'s ``SIGNING_KEY``, the value is the **Web Auth Domain**. It can be at most 64 characters.
```

Comment:
💡 I think it is worth splitting the source account, and value into separate dot points.